src/Raven.Studio/typescript/viewmodels/shardedReactViewModelBase.ts

```diff
@@ -33,6 +33,14 @@ abstract class shardedReactViewModelBase extends shardViewModelBase {
             location: this.location,
         };
 
+        if (typeof args === "string") {
+            reactProps["pathParam"] = args;
+        }
```

Comment:
What if there are multiple path params? e.g. `/users/:userId/orders/:orderId`

Reply:
It should be also handled in `reactViewModelBase.ts`

Reply:
Now the args are always spreaded (look at reactProps). We should remove that.
You will need to check the views that use query and path params and adjust them accordingly to these changes. Let's also add types to the props.